introduction/java_loops_II

```diff
@@ -0,0 +1,25 @@
+import java.util.*;
+import java.io.*;
+import java.lang.Math;
+
+class Solution{
+  public static void main(String []argh) {
+    Scanner in = new Scanner(System.in);
+      int t = in.nextInt();
+      int result = 0;
+      int exponent = 2;
+        for(int i = 0;i < t;i++){
```

Comment:
For loops with a body longer than 3 lines use meaningful iterator name, i. e. rename i.